client/reader/following-manage/index.jsx

```diff
@@ -123,6 +123,11 @@ class FollowingManage extends Component {
 
 export default connect(
 	( state, ownProps ) => ( {
-		searchResults: getReaderFeedsForQuery( state, ownProps.sitesQuery ) || [],
+		// the feed searching gives different urls than /following/mine.
+		// therefore we decide to omit it and use the feed_URL from the feed/site
```

Comment:
this is because the API is busted. It should be reliably returning a `subscribe_URL`, but we missed a case. I can fix it up.